assets/src/edit-story/app/rightClickMenu/provider.js

```diff
@@ -84,6 +98,14 @@ function RightClickMenuProvider({ children }) {
     DEFAULT_RIGHT_CLICK_MENU_STATE
   );
 
+  const selectedElement = selectedElements?.[0];
+  const currentPosition = currentPage?.elements.findIndex(
+    (element) => element.id === selectedElement?.id
+  );
+  const canElementMoveBackwards = currentPosition > 1;
+  const canElementMoveForwards =
+    currentPosition < currentPage?.elements.length - 1;
```

Comment:
should we disable both of these if the selected element is a background element?

Reply:
background images don't get the option to move forward or backwards so this should be ok.

<img width="817" alt="Screen Shot 2021-07-22 at 10 00 24 AM" src="https://user-images.githubusercontent.com/10720454/126678944-a9f9cc73-4fe0-476e-b20e-e12b347451f3.png">

Reply:
Yeah this menu only shows if you click on media that isn't background 😄